core/ir/aarch64/codec.c

```diff
@@ -1594,8 +1594,14 @@ decode_opnd_fpimm8(uint enc, int opcode, byte *pc, OUT opnd_t *opnd)
      *     = imm64:imm64 (Q=1)
      */
     union {
+#ifdef DR_HOST_AARCH64
```

Comment:
I think it would be nicer to have this be `#ifdef HAVE_HALF_FLOAT`, which matches other platform capabilities.  That define could be like these, set from cmake and configure.h:
```
make/configure.cmake.h:#cmakedefine HAVE_FVISIBILITY
make/configure.cmake.h:#cmakedefine HAVE_TYPELIMITS_CONTROL
make/configure.cmake.h:#cmakedefine HAVE_RSEQ
```

Reply:
Re-iterating request to have a capability rather than a hardcoded arch.

Reply:
Done.